### PR TITLE
Ship spawn adjustments

### PR DESCRIPTION
**Pull request by lectronyx**
## About The Pull Request

Couple things are done here. One, PGF is disabled, as is SRM for the time being. Two, I've started some adjustments on the Aegis, which will ultimately become a Cybersun ship. Another ship, of course, is being made Roseus.

## Why It's Good For The Game

We're taking out these 3 factions - PGF, SRM, and SUNS - due to not fitting with the lore. That said, the SRM ship's been adjusted so that some of the text better fits the Moffic Church that Ovilum's working on. I'm hesitant to do much there, since I want him to have more input on his faction's ship. It will become playable again once the church is better established, though I'm leaving the ball largely in his court.

Our shift away from the Sarathi leaves the PGF in a weird spot, and I don't really want to keep them around in my lore. They're Space America, and frankly, that's what SolFed's usually for. Since we want to be more approachable for people used to regular SS13 lore, giving SolFed's gimmick away to someone else feels like change for the sake of being different and not better - that's not what we're after at all. Maybe this ship will eventually be given a SolFed coat of paint once that whole mess gets sorted out, but for now it's being made non-playable in order to help avoid accidental lore violations.

SUNS is being removed, with Cybersun being the medical faction. SUNS, here, has a really weird position it's stuck in because it treads on the aesthetic we want for Roseus Galactic while sharing functionality with Cybersun. While one could try to argue Roseus Galactic is in the same situation, this is ultimately false - while Cybersun has one ship that touches on Roseus' role, they don't have any emphasis on it at all. Cybersun is otherwise, overwhelmingly, a medical faction - their other ship is medical, all of their ads are medical, they fit this slot very well. Roseus Galactic was already credited for the Infiltrator Suit in /TG/code (via the description in the uplink), which carried over to Shiptest due to people not paying much attention to the contents of their code. In our lore, Roseus Galactic is the intelligence and espionage faction - saboteurs and basic agents at their loudest. These are spies and celebrities, both actors in their own rights, as well as the staff that takes care of them. Makeup artists, security, etc. With that in mind, it only makes sense that the ship focused on infiltration and deception be given to the faction focused on intelligence gathering and acting. SUNS aesthetic is being given to Roseus Galactic because they are the least redundant of the two factions. If we ever need an additional medical faction, we'll most likely bring over Interdyne Pharmaceuticals - however, for now, what we have is enough. SUNS is only taking one faction's aesthetic while competing with another faction over the same job and not enough to really set them apart in any interesting way.

TL;DR: Lorebreaking ship is aspawn only, Moffic Church ship aspawn only until the faction gets more writing, SUNS is being disbanded with its Aegis being given to Cybersun, Roseus Galactic is gaining a little more presence by acquiring a Cybersun ship that doesn't fit the faction.

## Changelog

:cl:
del: PGF ship made aspawn. BYEBYE. BYEBYE.
del: SRM ship made aspawn until further notice.
tweak: Aegis made Cybersun
tweak: Kansatsu made Roseus
/:cl:

### CODE REVIEW

**Comment by lectronyx** [comment]
Incomplete, redoing

**Comment by lectronyx** [comment]
wait! I never needed to close this at all, what was I thinking?

Cybersun ship names, or at least the Kansatsu, seem to be japanese words relevant to the ship's function. Seiiki is, according to Google Translate, Japanese for "Sanctuary." Works pretty well for the Aegis!

Roseus Galactic ships are going to follow the naming scheme of stage/performance terms and things like that, however relevant they can be. The Soundcheck-class is such because the soundcheck is done hours before the performance itself - and a Soundcheck-class ship is scouting ahead to gather information before any action can be taken. I think that makes sense!
